hassrelease/changelog.py

```diff
@@ -14,7 +14,9 @@
 IGNORE_LINE_LABELS = set(['reverted'])
 LABEL_HEADERS = {
     'new-platform': 'New Platforms',
+    'new-feature': 'New Features (incomplete)',
```

Comment:
Don't add the words `incomple`, it means I will have to remove that every time I run the generator.